app/components/donation/donation-container.js

```diff
@@ -0,0 +1,14 @@
+import Ember from 'ember';
+
+const {
+  Component,
+  computed: { bool }
+} = Ember;
+
+export default Component.extend({
+  classNames: ['donation-container'],
+  donationAmount: 0,
+  projectTitle: null,
+
+  canDonate: bool('projectTitle')
```

Comment:
This just returns true if there is a `projectTitle` and false if there isn't.

Technically, we could do `{{#if projectTitle}}` but I think `{{#if canDonate}}` better explains there scenario--the user cannot make a donation, as no project has been selected.  It's a it of additional code, but I think it makes the template easier to read.

Let me know if you'd prefer the logic to be on the `projectTitle` explicitly